programs/mpl-core/src/plugins/plugin_registry.rs

```diff
@@ -21,8 +22,8 @@ pub struct PluginRegistryV1 {
     pub key: Key, // 1
     /// The registry of all plugins.
     pub registry: Vec<RegistryRecord>, // 4
-    /// The registry of all external, third party, plugins.
-    pub external_registry: Vec<ExternalRegistryRecord>, // 4
+    /// The registry of all adapter, third party, plugins.
+    pub external_plugin_adapter_registry: Vec<ExternalPluginAdapterRegistryRecord>, // 4
```

Comment:
I'm ok with this but for regular `plugin` we just say `RegistryRecord` without the name `plugin`, so by extension of that logic we could just still call these `external_registry` and `ExternalRegistryRecord`